ledger/ledger-api-test-tool/src/test/suite/scala/com/daml/ledger/api/testtool/NamesSpec.scala

```diff
@@ -27,8 +27,8 @@ class NamesSpec extends AnyWordSpec with Matchers {
     }
 
     "not be a prefix of any other name, so that each test can be included independently" in {
```

Comment:
```suggestion
    "not be a prefix of or equal to any other name, so that each test can be included independently" in {
```
```suggestion
    "not be a prefix of any other name, so that each test can be included independently" in {
```